app/views/device_stories/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= content_for(:title) do %>
+    <%= @device_story.last_location_name %>
+<%- end -%>
+
+<div class="span9">
+    <div class="row">
+        <h2>
+            <%= @device_story.last_location_name %> (<%= @device_story.device_urn %>)
+        </h2>
+
+        <div class="span2">
+            <table class="table table-hover">
+                <thead>
+                    <%= time_ago_in_words(@device_story.last_seen) %> ago (<%= @device_story.last_seen %>)
+                </thead>
+                <tr>
+                    <td style="font-size: 25px"><%= @device_story.last_values %></td>
+                </tr>
+            </table>
+            <a href="http://grafana-dev.safecast.cc/d/DFSxrOLWk/safecast-device-details?orgId=1&from=now-90d&to=now&refresh=15m&var-device_urn=<%= @device_story.device_urn %>"
```

Comment:
We should swap this out for a configurable value, and set the default to https://grafana.safecast.cc/d/DFSxrOLWk